### PR TITLE
[SD-1329] Idle Channel notification

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: client/app/scripts/superdesk-ingest/module.js [hunk 1]
```diff
@@ -217,6 +217,7 @@ define([
             link: function($scope) {
 
                 var DEFAULT_SCHEDULE = {minutes: 5, seconds: 0};
+                var DEFAULT_IDLE_TIME = {hours: 0, minutes: 0, seconds: 0};
```
Comment (amagdas) [inline client/app/scripts/superdesk-ingest/module.js]: Do we need to support seconds? Having just minutes should be enough I think..

Comment (marwoodandrew) [inline client/app/scripts/superdesk-ingest/module.js]: You're probably right, seconds are over kill!

**Comment by amagdas** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**